cli/bblfshctl/cmd/driver_install.go

```diff
@@ -21,17 +21,17 @@ var (
 		"python":     "docker://bblfsh/python-driver:latest",
 		"java":       "docker://bblfsh/java-driver:latest",
 		"bash":       "docker://bblfsh/bash-driver:latest",
-		"php":        "docker://bblfsh/php-driver:latest",
 		"javascript": "docker://bblfsh/javascript-driver:latest",
 		"ruby":       "docker://bblfsh/ruby-driver:latest",
 		"typescript": "docker://bblfsh/typescript-driver:latest",
 	}
-
+	// RecommendedDriver is the list of drivers in beta state or better
```

Comment:
As a note for myself: this list should be generated the same way as https://github.com/bblfsh/documentation/issues/135